src/gluonts/zebras/_period.py

```diff
@@ -177,6 +177,9 @@ def to_pandas(self):
     def to_timestamp(self):
         return self.data.astype(object)
 
+    def epoch(self) -> int:
```

Comment:
Minor, but

```suggestion
    def unix_epoch(self) -> int:
```

may be less ambiguous. (I took a second to understand what this is)